### PR TITLE
Add app config for djmoney.contrib.exchange

### LINKED ISSUE #684: Migration created after makemigrations with DEFAULT_AUTO_FIELD modified settings
An migration called "0002_alter_rate_id.py" created after running "makemigrations" with
`DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'`

### PR DESCRIPTION
**Pull request by ventur40**
Fixes #684

### CODE REVIEW

**Review by benjaoming — CHANGES_REQUESTED** [review]
Thanks, it needs to be called `apps.py` to work :)

**Review by benjaoming — APPROVED** [review]
LGTM! Thanks for fixing :rocket:

File: djmoney/contrib/exchange/apps.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class ExchangeConfig(AppConfig):
```
Comment (stianjensen) [inline djmoney/contrib/exchange/apps.py:1]: Looks like you're missing an import:
`from django.apps import AppConfig`

Comment (ventur40) [reply djmoney/contrib/exchange/apps.py:4]: Added!

**Comment by alexnorgaard** [comment]
I can see that this is failing on some tests, but the logs are too old to view. Can it be re-run I can have a look? If it needs more  work, I will be happy to give it a go, as I need this fix for implementing django-money on my company's existing code base.

**Comment by stianjensen** [comment]
Users with the correction permission should have access to a button like this on the top-right corner of the summary tab for checks:
<pre align="center"></pre>

**Comment by benjaoming** [comment]
Doesn't exist and I should have all rights necessary :/

image

**Comment by alexnorgaard** [comment]
> Doesn't exist and I should have all rights necessary :/
> 
> image

Not even if you click the Latest #2 button?

**Comment by benjaoming** [comment]
Nope, it's a drop-down.. I've tried _everything_ afaict
image

**Comment by stianjensen** [comment]
https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs

Aha, re-run is only available up to 30 days after the first run of a job.

**Comment by benjaoming** [comment]
Nice observation @stianjensen ! It'd be nice of those UI people working at GitHub to just disable the button rather than completely removing it :)

**Comment by alexnorgaard** [comment]
I have opened a new PR with the same changes here: https://github.com/django-money/django-money/pull/715 
The workflow needs to be approved

**Pull request closed without merging**